core/src/main/java/io/crate/module/CrateCoreModule.java

```diff
@@ -53,6 +54,11 @@ public CrateCoreModule(Settings settings, IndexEventListenerProxy indexEventList
                 "but have not configured a license. Please request a license or deactivate the "+
                 "Enterprise Edition. https://crate.io/enterprise");
         }
+        if (CrateRestFilter.ES_API_ENABLED_SETTING.get(settings)) {
+            logger.warn("You have currently enabled the Elastic Search API, " +
```

Comment:
how about: "The Elasticsearch API is currently enabled, which makes it possible to request sensitive information, such as user data."

though this implies there is other sensitive data? is that true? or should we just trim it to say "which makes it possible to request user data"?